SignalServiceKit/src/Network/API/Requests/TSRequest.h

```diff
@@ -14,4 +10,9 @@
 
 - (void)makeAuthenticatedRequest;
 
+#pragma mark - Factory methods
+
+// move to builder class/header
++ (instancetype)setProfileNameRequestWithProfileName:(NSString *)encryptedName;
```

Comment:
whoops - good catch. I opted for a separate Builder class instead. I'll remove this declaration.